frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelConversionWindowFilter.tsx

```diff
@@ -25,49 +25,44 @@ export function FunnelConversionWindowFilter(): JSX.Element {
     const intervalBounds = TIME_INTERVAL_BOUNDS[conversionWindow.funnel_window_interval_unit]
 
     return (
-        <div className="funnel-options-conversion-window">
-            <span className="funnel-options-conversion-window-label">
-                Conversion window limit{' '}
-                <Tooltip
-                    title={
-                        <>
-                            <b>Recommended!</b> Limit to users who converted within a specific time frame. Users who do
-                            not convert in this time frame will be considered as drop-offs.
-                        </>
-                    }
-                >
-                    <InfoCircleOutlined className="info-indicator" />
+        <div style={{ marginTop: 16 }}>
+            <Row align="middle">
+                Conversion window
+                <Tooltip title="Only conversions within this timeframe of performing the initial step will be counted.">
```

Comment:
Nit: I'm not sure if it's completely explicit (without you having to give it deeper thought) that users will be counted up to the last step they did but then considered as drop-offs.